include/libstatistics_collector/topic_statistics_collector/received_message_age.hpp

```diff
@@ -120,6 +123,24 @@ class ReceivedMessageAgeCollector : public TopicStatisticsCollector<T>
     }
   }
 
+  /**
+  * Handle a new incoming message. Calculate message age if a valid Header is present.
+  *
+  * @param received_message the message to calculate age of.
+  */
+  void OnMessageReceived(
+    const rmw_message_info_t & received_message) override
```

Comment:
We'll want to add an explicit package.xml/CMakeLists.txt dependency on `rmw` for this.